content/news/_posts/2018-03-30-google-deepvariant.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Can a \"Google AI\" Build Your Genome Sequence?"
+excerpt: "A new AI-powered tool from Google promises more-accurate genome sequences, but its impact on genomics research remains to be seen"
+categories: news
+author: alex_constantino
+editor: andrey_kurenkov
+tags: [biology,google,genomics]
+comments: true
+share: true
+published: true
+---
+
+## What Happened
+
+In December 2017, Google [announced a new tool called DeepVariant](https://research.googleblog.com/2017/12/deepvariant-highly-accurate-genomes.html):
+
+> Today, we announce the open source release of DeepVariant, a deep learning technology to reconstruct the true genome sequence from HTS sequencer data with significantly greater accuracy than previous classical methods.
+
+Simultaneously, the DeepVariant authors published [a paper](https://www.biorxiv.org/content/biorxiv/early/2018/03/20/092890.full.pdf) detailing their research and, unlike many research projects, provided the complete underlying [code](https://github.com/google/deepvariant) — yay! So what is it, and how does it work?
+
+First, let’s talk about DNA. DNA is the molecule that determines your genetic makeup, influencing everything from your height to your eye color to your risk of getting cancer. Each person’s genetic information can be represented as a sequence of the letters A, T, C, and G. Thanks to a technology called high-throughput sequencing, it is now possible to read the ~3 billion letters in your DNA for just ~$1000, which has in turn been fueling the rapidly growing field of genomics research.
+
+Here’s how high-throughput sequencing typically works: DNA molecules are isolated from blood or spit and broken into millions of pieces with a couple hundred letters each. To read each letter, a machine adds a chemical dye to each piece, lights up the dye with a laser, and takes a picture. The rest of the process happens on computers — a program processes each picture to determine the most likely letter sequence for each piece, aligns each piece to a complete reference sequence, and lists "variants" (a variant is any difference between the sample and reference sequence e.g. “three letters deleted” or “A instead of T”).
+
+<figure>
+	<img src="{{site.url}}/content/news/images/2018-03-30-google-deepvariant/image_0.png" alt="Pileup Examples">
+	<figcaption>
+Examples of the "pile-up" pictures that DeepVariant interprets. The different colors correspond to the A’s, T’s, G’s, and C’s of DNA. See those gaps in sections B and C? Those indicate that part of the genome was deleted.
```

Comment:
Depending on how formal you want to be, drop the "see those gaps" and just say "The gaps in sections B and C indicate deleted genome sections" or something.